blocks/init/src/Blocks/components/video-button/video-button.php

```diff
@@ -37,5 +38,5 @@ class="<?php echo \esc_attr($videoButtonClass); ?>"
 	data-micromodal-trigger="<?php echo \esc_attr($videoButtonModalId); ?>"
 	aria-label="<?php echo \esc_html($videoButtonLabel); ?>"
 >
-	<?php echo \wp_kses_post($icon); ?>
+	<?php echo $manifest['icon']; ?>
```

Comment:
Why is escaping removed? I see that other SVG icons in other components are escaped.

Reply:
in general there is no need to filter icons that we control and are stored in the manifest.